input/data/animal/example_freestall_animal.json

```diff
@@ -10,7 +10,7 @@
 	"breed": "HO",
 	"parity_fractions": {
 		"1": 0.36,
-		"2": 0.262,
+		"2": 0.26,
```

Comment:
This is based off @tomhuhh's x-validation rule that all the parity fractions should sum to exactly 1.0 and before I made this change they summed to 1.002. Can revert back if it's not correct.